src/components/pages/SavedCases/SavedCases.js

```diff
@@ -39,6 +41,28 @@ function SavedCases({ savedCases, setSavedCases, deleteBookmark }) {
       ),
       width: '5%',
     },
+    {
+      title: 'Download Report',
+      render: (text, record) => (
+        <PDFExportButton
+          key="export"
+          fileName={`Judge_${record.last_name}_Case_${record.case_number}`}
+          caseData={[
+            {
+              case_id: record.case_number,
+              case_date: record.case_date,
+              judge_name: record.last_name,
+              case_origin: record.case_origin_city,
+              case_outcome: record.case_outcome,
+              country_of_origin: record.country_of_origin,
+              gender: record.gender,
+              type_of_violence: record.type_of_violence,
+            },
```

Comment:
Perhaps we can include the state in the Case Origin field?

We can use string interpolation to accomplish this on the `case_origin` property:
```
case_origin: `${record.case_origin_city}, ${record.case_origin_state}`
```

Reply:
Good idea. 
Once we get a handle on what should actually be in the reports, we can implement items like this.
UI/UX should be involved here as well to get a meaningful report generated.